e2e-tests/shared-test-assertions/currency-form-fields/index.js

```diff
@@ -34,7 +34,7 @@ export const assertCurrencyFormFields = ({
   hint,
   legend,
   expectedRedirectUrl,
-  hasSaveAndBack,
+  hasSaveAndBack = true,
```

Comment:
should they all have a save and back? if they do we could get rid of this condition?

Reply:
Yes, the last "save and back" is added in the next PR, which removes all instances of `hasSaveAndBack`: https://github.com/UK-Export-Finance/exip/pull/3129